scripts/scripts.js

```diff
@@ -92,6 +93,57 @@ function detectSidebar(main) {
   }
 }
 
+function buildSingleLightBox(element, isGallery) {
+  const picture = element.querySelector('picture');
+  const actualLightbox = document.createElement('div');
+  if (isGallery) element.parentNode.parentNode.append(actualLightbox);
+  else element.parentNode.append(actualLightbox);
```

Comment:
I personally would find easier to read the following, and it seems to be the general style used in the rest of the code. Maybe an eslint rule to that effect should be added?
```suggestion
  if (isGallery) {
    element.parentNode.parentNode.append(actualLightbox);
  } else {
    element.parentNode.append(actualLightbox);
  }
```

Reply:
fixed